### PR TITLE
Adding new units of measurement 

### PR DESCRIPTION
**Pull request by Peyman-N**

### CODE REVIEW

File: instances/unitOfMeasurement/hertz.jsonld [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/unitOfMeasurement/hertz",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/UnitOfMeasurement",
+  "definition": "Hertz is the unit of frequency in the International System of Units (SI), equivalent to one event (or cycle) per second.",
```
Comment (lzehl) [inline instances/unitOfMeasurement/hertz.jsonld]: The hertz is ... AND please add the source of that definition (here is is Wikipedia).

File: instances/unitOfMeasurement/kilohertz.jsonld [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/unitOfMeasurement/kilohertz",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/UnitOfMeasurement",
+  "definition": "A unit of frequency equal to one thousand cycles per second; one thousand hertz.",
```
Comment (lzehl) [inline instances/unitOfMeasurement/kilohertz.jsonld]: please match form of definition used for hertz

File: instances/unitOfMeasurement/kilovolt.jsonld [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/unitOfMeasurement/kilovolt",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/UnitOfMeasurement",
+  "definition": "One thousand volts.",
```
Comment (lzehl) [inline instances/unitOfMeasurement/kilovolt.jsonld]: please match the definition form you used for hertz/kilohertz (here naturally vold / kilovolt)

File: instances/unitOfMeasurement/microvolt.jsonld [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/unitOfMeasurement/microvolt",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/UnitOfMeasurement",
+  "definition": "One millionth (10^-6) of a volt.",
```
Comment (lzehl) [inline instances/unitOfMeasurement/microvolt.jsonld]: please adapt (see previous comments)

File: instances/unitOfMeasurement/nanovolt.jsonld [hunk 5]
```diff
@@ -0,0 +1,16 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/unitOfMeasurement/nanovolt",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/UnitOfMeasurement",
+  "definition": " one billionth (10^-9) of a volt.",
```
Comment (lzehl) [inline instances/unitOfMeasurement/nanovolt.jsonld]: correct (unnecessary space) and see above

**Review by lzehl — CHANGES_REQUESTED** [review]
We should discuss conventions here. I'm unsure if my suggestions are the best solution for definitions.

**Comment by Peyman-N** [comment]
I completely agree we should standardize the definition all of the SI prefixes. 

I would suggest this definition :

A unit of `measurand` equal to `amount of` `main unit`.
Example:
A unit of **electric potential** equal to **_one thousand_** **volts**.

Or we can go the hard way and separate the SI prefixes from the unit, this can be beneficial in long run but would need a clean up; so I am against it.

**Comment by apdavison** [comment]
> A unit of `measurand` equivalent to `amount of` `main unit`.

"equal to", not "equivalent to"

File: instances/unitOfMeasurement/kilovolt.jsonld [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+  "@type": "https://openminds.ebrains.eu/controlledTerms/UnitOfMeasurement",
+  "definition": "A unit of electric potential equal to one thousand volts.",
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "kilovolt",
+  "preferredOntologyIdentifier": "http://purl.obolibrary.org/obo/UO_0000248",
+  "synonym": [
+    "kV",
+    "KV"
```
Comment (apdavison) [inline instances/unitOfMeasurement/kilovolt.jsonld]: I don't think we should list such non-standard forms as "KV"

**Review by apdavison — CHANGES_REQUESTED** [review]
one small change: remove "KV" from synonyms

**Review by lzehl — APPROVED** [review]
under the condition to remove KV from synonyms of kilovolt as @apdavison suggests.

**Review by apdavison — APPROVED** [review]

**Pull request merged**